Course/Course/Presentation/Subviews/CustomDisclosureGroup.swift

```diff
@@ -126,8 +127,9 @@ struct CustomDisclosureGroup: View {
                                                 VStack(alignment: .leading) {
                                                     HStack {
                                                         if sequential.completion == 1 {
-                                                            CoreAssets.finishedSequence.swiftUIImage
+                                                            CoreAssets.finishedSequence.swiftUIImage.renderingMode(.template)
```

Comment:
Is there a way to prevent this warning while following the specified guidelines?
_Line Length Violation: Line should be 120 characters or less; currently it has 125 characters_

Reply:
Unfortunately, didn't get a way to break it into multiple lines.